values-10k-to-1M.yaml

```diff
@@ -0,0 +1,32 @@
+# Minimal settings for scale of 10k to 1M events/month.
```

Comment:
What's the purpose of these files? As-is users will not look here.

I suggest writing a proper README instead (or another doc and linking it from README), outlining the configuration options etc.

Reply:
I haven't updated the readme, but I was planning to do that. The purpose is to make it easier and faster/easier/less error prone for the users for the initial install, will help increase engagement/activation and reduce support load. 

To elaborate the thinking was: as a user I have no idea how much resources various parts in Posthog need, but I do know how many events / month I get. If based on the latter we direct them to reasonably working config that would be great. 

In the beginning we're not going to be good (well I'm not yet Karl and James might be) at suggesting what the config should be, but over time with more testing and with the feedback from the community we'll get to a good config and then it's possible to get to very small install times. 

If we have a large values file with all possible configs, it's much harder for a new person to know what they need to be doing hence it will take more time (we can use comments like "you must set this" etc, but I think it's easier for our users with these short files). Files rather than only text/explanation in a readme is another attempt to reduce time cost for the initial install. 

I might be wrong about all of this, but this is what my intuition is telling me.